python_expression_dsl/evaluator.py

```diff
@@ -20,7 +20,7 @@
     Visitor,
 )
 from .parser import DSLParser
-
+import types
```

Comment:
@vbessonov Do you mind looking over this file I updated to make it work in python 3? One test was failing because it was looking for a `module` type as a built in class. Then, in line 366, `im_class` is not available anymore so I updated that to check `.__self__.__class__`.

Reply:
@EdwinGuzman, I checked it and it seems it works correctly, thanks!